test/mix/tasks/phoenix.gen.channel_test.exs

```diff
@@ -76,4 +76,10 @@ defmodule Mix.Tasks.Phoenix.Gen.ChannelTest do
       Mix.Tasks.Phoenix.Gen.Channel.run ["Admin.Room", "rooms", "new_message"]
     end
   end
+
+  test "name can't already be defined" do
+    assert_raise Mix.Error, fn ->
+      Mix.Tasks.Phoenix.Gen.Channel.run ["Channel", "channels"]
```

Comment:
Great, that's what I was thinking but I wasn't sure if it was kosher.  Appreciate all your feedback @josevalim, dziękuję!